util/winutil/s4u/s4u_windows.go

```diff
@@ -157,6 +157,7 @@ func Login(logf logger.Logf, srcName string, u *user.User, capLevel CapabilityLe
 			return nil, err
 		}
 		sessToken = dupToken
+		token.Close()
```

Comment:
The return on line 170 will call token to be Closed twice. Is that okay?

Reply:
You might want to do:

```go
   onceClose := sync.OnceFunc(func() { token.Close() })
   defer func() {    if err != nil { onceClose() } }
   ...
   onceClose()
````

Reply:
yep, that’s not okay because of the implementation of `golang.org/x/sys/windows.Token.Close()`. It is very unfortunate that it doesn’t reset the token handle value upon closing, and doesn't check whether the handle is 0, making the `Close()` method unsafe to call multiple times..